src/action_state.rs

```diff
@@ -522,7 +522,7 @@ pub struct ActionStateDriver<A: Actionlike> {
 ///
 /// This struct is principally used as a field on [`ActionData`],
 /// which itself lives inside an [`ActionState`].
-#[derive(Debug, Clone, PartialEq, Default, Serialize, Deserialize)]
+#[derive(Debug, Clone, PartialEq, Eq, Default, Serialize, Deserialize)]
```

Comment:
@alice-i-cecile I threw a couple `Eq` derives in here to satisfy clippy, do you know if that's a SemVer breaking change? I don't think it is, but I'm not sure if there's a scenario I'm not thinking of.

Reply:
Implementing new traits isn't semver breaking (which is why we can't have negative trait bounds). All good here!